src/components/Editor.test.tsx

```diff
@@ -23,28 +24,32 @@ const authenticatedSession = {
   status: "authenticated",
   unlayerUser: {
     id: unlayerUserId,
-    signature: unlayerUserSignature,
-  },
+    signature: unlayerUserSignature
+  }
 };
 
 describe(Editor.name, () => {
+  const setEditorStateMock = (state: EditorState) => {
```

Comment:
@andresmoschini que piensas de colocar este mock a ese nivel para usar ese comportamiento por defecto compartido entre las pruebas. Cuando se necesite un comportamiento mas especializado podríamos ponerlo dentro e cada _it()_

Reply:
No tengo opinión.

Tengo dudas sobre lo que hace:

```
state.unlayer?.loadDesign(sampleDesign);
```

Es necesario? si eso no fuera necesario podríamos simplemente usar `() => {}` y listo, o definir `const noop = () => {}` y usar eso.

Por otro lado, por ahí yo estoy equivocado, pero suelo usar la palabra `mock` para los dobles que tienen funcionalidad para inspeccionar lo que pasa con ellos, por ejemplo `jest.fn()`.

Para este tipo de doubles, prefiero usar simplemente el nombre, por ejemplo:

```suggestion
  const setEditorState = (state: EditorState) => {
```

O las palabras `double` o `fake`:

```suggestion
  const setEditorStateDouble = (state: EditorState) => {
```

De cualquier manera, no tengo posición fuerte y de a poco iremos encontrando un estilo consistente entre ambos.

Reply:
Pero en este momento, no tenés nada que inspeccionar, para qué vas a usar `jest.fn()`?

Reply:
Use el noop() que me recomendaste en un inicio y el jest.fn() en donde comprobé el paso por alguna función.